website/docs/docs/get-started/connect-your-database.md

```diff
@@ -149,12 +149,40 @@ As an end user, if your organization has set up BigQuery OAuth, you can link a p
 <Lightbox src="/img/docs/dbt-cloud/dbt-cloud-enterprise/gsuite/bq_oauth/bq_oauth_as_user.gif" title="Link Button in dbt Cloud Credentials Screen" />
 
 ## Connecting to Databricks
+We have two supported adapters to connect to Databricks: [dbt-databricks](/connect-your-database#dbt-databricks) and [dbt-spark](/connect-your-database#dbt-spark). For accounts on dbt 1.0 or later, we recommend using the dbt-databricks adapter. The dbt-databricks adapter is maintained by the Databricks team and is verified by dbt Labs. The Databricks team is committed to supporting and improving the adapter over time, so you can be sure the integrated experience will provide the best of dbt and the best of Databricks.
```

Comment:
@schottj I'm curious if the dbt-spark adapter maintained by the community? We might want to link to the [verified adapters](https://docs.getdbt.com/docs/supported-data-platforms#verified-adapters) section of the supported data platforms page. It looks like both are verified, is that not correct anymore?

Reply:
dbt-spark will continue to be maintained by dbt Labs.

We do want to recommend people to use the new one, yes.